### PR TITLE
R canopy

### PR DESCRIPTION
**Pull request by vinjana**
* [x] I have read the [guidelines for bioconda recipes](https://bioconda.github.io/guidelines.html).
* [ ] This PR adds a new recipe.
* [x] AFAIK, this recipe **is directly relevant to the biological sciences** (otherwise, please submit to the more general purpose [conda-forge channel](https://conda-forge.org/docs/)).
* [x] This PR updates an existing recipe.
* [ ] This PR does something else (explain below).

### CODE REVIEW

File: recipes/r-canopy/meta.yaml [hunk 1]
```diff
@@ -1,39 +1,95 @@
...
+
+  # You can also put a file called run_test.py, run_test.sh, or run_test.bat
+  # in the recipe that will be run at test time.
+
+  # requires:
+    # Put any additional test requirements here.
 
 about:
-  home: https://github.com/yuchaojiang/Canopy
+  home: https
```
Comment (jenzopr) [inline recipes/r-canopy/meta.yaml]: The value of `home` somehow got lost..?

File: recipes/r-canopy/meta.yaml [hunk 2]
```diff
@@ -1,39 +1,95 @@
...
+    ratios between the tumor and matched normal sample for somatic copy number alterations
+    (CNAs). These quantities can be directly taken from the output of existing software.
+    Canopy provides a  general mathematical framework for pooling data across samples
+    and sites to  infer the underlying parameters. For SNAs that fall within CNA regions,
+    Canopy infers their temporal ordering and resolves their phase.  When there are  multiple
+    evolutionary configurations consistent with the data, Canopy outputs  all configurations
+    along with their confidence assessment.
+  license_family: GPL2
+
+# The original CRAN metadata for this package was:
```
Comment (jenzopr) [inline recipes/r-canopy/meta.yaml]: Also remove comments from here on.

File: recipes/r-canopy/meta.yaml [hunk 3]
```diff
@@ -1,39 +1,95 @@
...
     - r-scatterplot3d
 
 test:
   commands:
-    - $R -e "library('Canopy')"
+    # You can put additional test commands to be run here.
+    - $R -e "library('Canopy')"  # [not win]
+    - "\"%R%\" -e \"library('Canopy')\""  # [win]
+
+  # You can also put a file called run_test.py, run_test.sh, or run_test.bat
```
Comment (jenzopr) [inline recipes/r-canopy/meta.yaml]: You can safely remove all unnecessary comments from the meta.yaml and the build.sh

Comment (jenzopr) [reply recipes/r-canopy/meta.yaml]: But of course not the `# [win]` directives, those are needed by conda

File: recipes/r-canopy/meta.yaml [hunk 4]
```diff
@@ -46,15 +43,7 @@ requirements:
 
 test:
   commands:
-    # You can put additional test commands to be run here.
-    - $R -e "library('Canopy')"  # [not win]
```
Comment (jenzopr) [inline recipes/r-canopy/meta.yaml]: This line has to stay - it's used for testing on non-windows systems.

File: recipes/r-canopy/meta.yaml [hunk 5]
```diff
@@ -46,15 +43,7 @@ requirements:
...
-    - "\"%R%\" -e \"library('Canopy')\""  # [win]
```
Comment (jenzopr) [inline recipes/r-canopy/meta.yaml]: This line has to stay as well - it's used for testing on windows systems ;)

File: recipes/r-canopy/meta.yaml [hunk 6]
```diff
@@ -46,15 +43,7 @@ requirements:
...
-
-  # You can also put a file called run_test.py, run_test.sh, or run_test.bat
-  # in the recipe that will be run at test time.
-
-  # requires:
-    # Put any additional test requirements here.
+    - $R -e "library('Canopy')"
```
Comment (jenzopr) [inline recipes/r-canopy/meta.yaml]: This wouldn't be needed (see above).

File: recipes/r-canopy/meta.yaml (hunk 4 above)
Comment (jenzopr) [reply recipes/r-canopy/meta.yaml]: Sorry if I wasn't clear in my last comments on the `# [win]` / `# [not win]` part..

File: recipes/r-canopy/meta.yaml (hunk 6 above)
Comment (vinjana) [reply recipes/r-canopy/meta.yaml]: O.k. I got the impression that it's not going to run on win. I re-added the directives and removed the comments.

**Comment by jenzopr** [comment]
Good job, thank you! I just tested the installation and `1.2.0` is already available. You can merge after the checks have passed. Just remember that the `1.2.0` will only be available with `conda install -c bioconda r-canopy==1.2.0` once the version `1.3.0` is in the repository.

**Comment by vinjana** [comment]
Thanks, Jens, for your support!

**Pull request merged**